packages/react/src/hooks/actions/testModels/todo.ts

```diff
@@ -12,6 +12,9 @@ export class Todo {
   readonly updatedAt?: string;
   constructor(init: ModelInit<Todo, TodoMetaData>) {
     this.name = init.name;
+    // this is only a test model and this is to make TS happy about
+    // the id attribute being set on the constructor
+    this.id = Date.now().toString();
```

Comment:
Someone who knows DataStore / TS better than I do, is this fine? TS was complaining:

<img width="1063" alt="CleanShot 2023-02-02 at 09 32 03@2x" src="https://user-images.githubusercontent.com/321279/216398904-afe23d8f-9878-4c3e-9ffe-e6023f565c6c.png">

Reply:
If `id` is something that's always present through DataStore actions, you can add a `!` post-fix:

```ts
readonly id!: string
```

But someone who knows DataStore better than I do can check if this is okay 😄

Reply:
Works! Thanks!